medical_ui/medical_ui/settings.py

```
@@ -0,0 +1,80 @@
+    value = os.getenv(name)
+    if value is None:
+        return default
+    return [item.strip() for item in value.split(",") if item.strip()]
+
+
+DEBUG = _env_bool("DJANGO_DEBUG", ENV != "prod")
+
+SECRET_KEY = os.getenv("DJANGO_SECRET_KEY")
+if not SECRET_KEY:
+    if DEBUG:
+        SECRET_KEY = "django-insecure-dev-only-key"
```

Ruff (0.15.7)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)